### PR TITLE
Move multi node readme to m3coordinator

### PR DESCRIPTION
**Pull request by benraskin92**

### CODE REVIEW

File: benchmark/README.MD [hunk 1]
```diff
@@ -19,6 +19,46 @@ To benchmark m3db writes using m3coordinator.
 
         $ ./bin/m3dbnode -f $GOPATH/src/github.com/m3db/m3coordinator/benchmark/m3db_config.yaml
 
+4a) For a three-node m3db cluster on GCP:
+
+    1. Download necessary packages
+        $ sudo apt-get install golang golint make git golang-glide
+    2. Set GOPATH:
+        $ export GOPATH=/home/<user_name>/code
```
Comment (nikunjgit) [inline benchmark/README.MD]: GOPATH=$HOME/code

Comment (nikunjgit) [reply benchmark/README.MD]: same thing everywhere else

Comment (nikunjgit) [reply benchmark/README.MD]: also maybe just talk about steps and then list the commands separately so people can just copy paste the whole thing

Comment (arnikola) [reply benchmark/README.MD]: Would it make sense to put it into a script that just does the steps?

File: benchmark/README.MD [hunk 2]
```diff
@@ -19,6 +19,44 @@ To benchmark m3db writes using m3coordinator.
...
         $ ./bin/m3dbnode -f $GOPATH/src/github.com/m3db/m3coordinator/benchmark/m3db_config.yaml
 
+4a) For a three-node m3db cluster on GCP:
+
+    1. Download necessary packages
+        $ sudo apt-get install golang golint make git golang-glide
+    2. Setup paths and clone m3db
+        $ export GOPATH=$HOME/code
+        $ mkdir -p $HOME/code/src/github.com/m3db
+        $ git clone https://github.com/m3db/m3db
```
Comment (nikunjgit) [inline benchmark/README.MD]: cd ?

File: benchmark/README.MD [hunk 3]
```diff
@@ -19,6 +19,44 @@ To benchmark m3db writes using m3coordinator.
 
         $ ./bin/m3dbnode -f $GOPATH/src/github.com/m3db/m3coordinator/benchmark/m3db_config.yaml
 
+4a) For a three-node m3db cluster on GCP:
+
+    1. Download necessary packages
+        $ sudo apt-get install golang golint make git golang-glide
+    2. Setup paths and clone m3db
+        $ export GOPATH=$HOME/code
```
Comment (arnikola) [inline benchmark/README.MD]: Shouldn't this be 
```
export GOPATH=$HOME/gocode
mkdir -p $GOPATH/src/github.com/m3db
```

**Pull request merged**